event-sourcing/event-repository/event-repository-jdbc/src/test/java/uk/gov/justice/services/eventsourcing/repository/jdbc/event/EventJdbcRepositoryFactoryTest.java

```diff
@@ -54,4 +59,12 @@ public void shouldProduceEventJdbcRepository() throws Exception {
         final Logger loggerField = getValueOfField(eventJdbcRepository, "logger", Logger.class);
         assertThat(loggerField, is(notNullValue()));
     }
+
+//    @Test
+//    public void shouldProduceEventJdbcRepositoryWithDataSource() throws Exception {
+//
+//        final EventJdbcRepository eventJdbcRepository = eventJdbcRepositoryFactory.eventJdbcRepository(datasource);
+//
+//        assertThat(eventJdbcRepository, is(notNullValue()));
+//    }
```

Comment:
This commented out test should be reinstated or removed

Reply:
removed